src/uwtools/drivers/forecast.py

```diff
@@ -126,10 +133,71 @@ def resources(self) -> None:
         ???
         """
 
-    def run(self):
+    def run(self, log: Optional[Logger] = None) -> None:  # pragma: no cover
         """
         Runs FV3.
         """
+        # Set up logging.
+        if log is None:
+            name = f"{inspect.stack()[0][3]}"
+            log = cli_helpers.setup_logging(
+                log_file="/dev/stdout",
+                log_name=name,
+                quiet=False,
+                verbose=False,
+            )
+        # Read in the config file.
+        experiment_config = config.YAMLConfig(self._config_file)
```

Comment:
I'm confused by `experiment_config` because we've got an "experiment" concept, but here we're in a "forecast" context. Do we need a clarification of concepts, or is this ok?

Reply:
nope, you're right, I missed that